src/main/java/com/connexta/ddf/catalog/direct/ExtendedMethods.java

```diff
@@ -0,0 +1,179 @@
+package com.connexta.ddf.catalog.direct;
+
+import static com.connexta.jsonrpc.JsonRpc.INTERNAL_ERROR;
+import static com.connexta.jsonrpc.JsonRpc.INVALID_PARAMS;
+
+import com.connexta.jsonrpc.DocMethod;
+import com.connexta.jsonrpc.Error;
+import com.connexta.jsonrpc.MethodSet;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableMap.Builder;
+import ddf.catalog.CatalogFramework;
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.Result;
+import ddf.catalog.data.impl.AttributeImpl;
+import ddf.catalog.data.types.Core;
+import ddf.catalog.data.types.Security;
+import ddf.catalog.federation.FederationException;
+import ddf.catalog.filter.FilterBuilder;
+import ddf.catalog.operation.CreateRequest;
+import ddf.catalog.operation.CreateResponse;
+import ddf.catalog.operation.Query;
+import ddf.catalog.operation.QueryRequest;
+import ddf.catalog.operation.QueryResponse;
+import ddf.catalog.operation.impl.CreateRequestImpl;
+import ddf.catalog.operation.impl.QueryImpl;
+import ddf.catalog.operation.impl.QueryRequestImpl;
+import ddf.catalog.source.IngestException;
+import ddf.catalog.source.SourceUnavailableException;
+import ddf.catalog.source.UnsupportedQueryException;
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import org.opengis.filter.Filter;
+
+public class ExtendedMethods implements MethodSet {
```

Comment:
:pencil2: Could add JavaDoc to explain this is for methods that extend the catalog framework methods.